### ISSUE TITLE
encrypted VM images support

### ISSUE THREAD

**Issue by adrelanos** [grml/grml-debootstrap#131]
Would be nice if encryption was supported for VM images (using standard dm-crypt).

This is interesting since (VM) raw images created by grml-debootstrap can also potentially be written (`dd`) to USB or other hard drives.

The masterkey would be known to the public but users could use `cryptsetup-reencrypt` to fix that once booted for the first time.

**Comment by adrelanos** [comment]
It's possible. This guide explains using debootstrap on encrypted LVM.

http://www.coredump.us/index.php?n=Main.DebianEncryptedDebootstrap

**Comment by mika** [comment]
@adrelanos yeah, (full disk) encrypted systems are possible, but the way VM images are created is a bit more special, so that might not be entirely straight forward, though technically possible as well :)
